jcapi-systemusers.go

```diff
@@ -138,9 +141,18 @@ func getJCUserFieldsFromInterface(fields map[string]interface{}, user *JCUser) {
 	if _, exists := fields["password_date"]; exists {
 		user.PasswordDate = fields["password_date"].(string)
 	}
+
+	if _, exists := fields["password_expiration_date"]; exists {
+		var err error
+		user.PasswordExpirationDate, err = time.Parse(time.RFC3339, fields["password_expiration_date"].(string))
+		if err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
-func getJCUsersFromInterface(userInt interface{}) []JCUser {
+func getJCUsersFromInterface(userInt interface{}) ([]JCUser, error) {
```

Comment:
This signature change probably breaks a lot of the other examples. We need to update those. Same with function above